app/src/features/apiClient/screens/environment/components/VariablesListHeader/VariablesListHeader.tsx

```diff
@@ -78,8 +78,8 @@ export const VariablesListHeader: React.FC<VariablesListHeaderProps> = ({
           >
             Save
           </RQButton>
-          {exportActions.showExport && (
-            <RQButton type="primary" onClick={exportActions.onExportClick} disabled={!exportActions.enableExport}>
+          {exportActions?.showExport && (
```

Comment:
`showExport` is still required in the propType, so either also fix the types or (if the types already represent the correct logic for the props) write a more verbose check here

Reply:
same for all the other optional chaining added for the required args, just adds confusion to the code